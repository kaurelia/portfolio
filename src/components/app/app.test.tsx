import App from "./app";
import { render, waitFor } from "@testing-library/react";

describe("Test app component", () => {
  it("Check if component renders properly", async () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
