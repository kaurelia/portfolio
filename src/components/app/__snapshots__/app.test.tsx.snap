// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test app component Check if component renders properly 1`] = `
<div>
  LOADING...
</div>
`;
